experimental-default-global-styles.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file isn't included in the plugin:

https://plugins.trac.wordpress.org/browser/gutenberg/tags/7.7.1

Because we whitelist files to include in the ZIP, and this isn't one of them:

https://github.com/WordPress/gutenberg/blob/56a12a628b2b5ebab36fafd9cf5be185a56cdb07/bin/build-plugin-zip.sh#L112-L125

We _could_ simply add the pattern there, but I also think we should try to align some convention of where we want these sort of static files to live.

- https://github.com/WordPress/gutenberg/tree/master/lib/demo-block-template-parts
- https://github.com/WordPress/gutenberg/tree/master/lib/demo-block-templates
- https://github.com/WordPress/gutenberg/tree/master/lib/patterns

If we move this into `lib` or a subfolder, it would be included.

Reply:
[This PR](https://github.com/WordPress/gutenberg/pull/20530) moves it within `lib/global-styles`.